src/main/java/mate/academy/spring/entity/Rent.java

```diff
@@ -0,0 +1,85 @@
+package mate.academy.spring.entity;
+
+import java.time.LocalDate;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "rent")
+public class Rent {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "rent_id")
+    private Long id;
+    @Column(name = "rent_date")
+    private LocalDate rentDate;
+    @ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
why we use many to one here?
maybe better use one to one? What do you think?

Reply:
I use ManyToOne here, because, one User have many Rents, but we can use OneToOne

Reply:
fix